src/offset/local/tz_info/timezone.rs

```diff
@@ -50,6 +53,14 @@ impl TimeZone {
             }
         }
 
+        // ohos merge all file into tzdata since ver35
+        #[cfg(target_env = "ohos")]
```

Comment:
Is there any point in falling through after this? If so, please add a comment making explicit why boths are useful.